addons/binding/org.openhab.binding.lgtvserial/src/main/java/org/openhab/binding/lgtvserial/internal/protocol/serial/commands/ColorTemperatureCommand.java

```diff
@@ -0,0 +1,9 @@
+package org.openhab.binding.lgtvserial.internal.protocol.serial.commands;
+
+public class ColorTemperatureCommand extends BasePercentCommand {
```

Comment:
When compared to their channel definitions, the types of ColorTemperatureCommand and ColorTemperature2Command are reversed. I am able to get the ColorTemperatureCommand to work when changing it to a BaseStringCommand.